File: src/targets/clojure/clj_http/fixtures/application-form-encoded.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:form-params {:foo "bar"
                                                           :hello "world"}})
=======
(client/post "http://mockbin.com/har" {:form-params {:foo "bar"
                                                     :hello "world"}})
>>>>>>> upstream/master

File: src/targets/php/guzzle/fixtures/application-form-encoded.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
>>>>>>> upstream/master
  'form_params' => [
    'foo' => 'bar',
    'hello' => 'world'
  ],
  'headers' => [
    'content-type' => 'application/x-www-form-urlencoded',
  ],
]);

echo $response->getBody();